src/FeaturesViewer.js

```diff
@@ -5,7 +5,7 @@ var _ = require("underscore");
 var d3 = require("d3");
 
 var Constants = require("./Constants");
-var DataLoader = require("./DataLoader");
+var DataLoader = require("./dataLoader");
```

Comment:
Could you rename _dataLoader_ to _DataLoader_ instead for consistency? The change must have been lost during a merge...